hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - CreateKey Metrics.json

```diff
@@ -15,18 +15,319 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": null,
   "links": [],
   "liveNow": false,
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
A minor nit: The remaining panels of the Create Key Dashboard appear to have their panels collapsed by default. Do we either keep this as `true` or modify the remaining occurrences as `false` (to sort of be consistent)?

Reply:
Done